README.md

```diff
@@ -278,6 +278,8 @@ $.mockjax({
     var authorID = settings.urlParams.authorID;
     var isbnNumber = settings.urlParams.isbnNumber;
     // etc...
+    //gets the request data/body sent via ajax
+    var requestBody = settings.data 
```

Comment:
This isn't exactly true. The `data` on the settings object is whatever the user passed in and is not actually a request body. I'm ok with including some info in the README, but let's rename that variable and change the comment to be more clear about the request not being made and thus there not being a request body.